git_hg_sync/repo_synchronizer.py

```diff
@@ -158,7 +181,19 @@ def sync(
             except Exception as e:
                 raise RepoSyncError(tag_operation, e) from e
 
+            tag_branches_to_push.add(tag_operation.tags_destination_branch)
+
+        for tag_branch in tag_branches_to_push:
+            push_args.append(f"{tag_branch}:refs/heads/branches/{tag_branch}/tip")
+
         logger.debug(f"Push arguments: {push_args}")
+
+        if len(push_args) < 2:
+            logger.warning(
+                "No explicit references to push resulted from processing this message."
+            )
+            return
```

Comment:
nit: This warning message is not very clear. It looks like `push_args` always has at least one item, so maybe this should be `if len(push_args) == 1` and the warning should say something more like `insufficient push arguments"?